recipe/build.sh

```diff
@@ -25,5 +25,11 @@ fi
 
 make
 make install
-ln -s $PREFIX/bin/python3.5 $PREFIX/bin/python
-ln -s $PREFIX/bin/pydoc3.5 $PREFIX/bin/pydoc
+
+python -c "import os; print('.'.join(os.environ['PKG_VERSION'].split('.')[:2]))" > temp.txt
```

Comment:
We're using `python` above for the branding. What's the difference if we use it here?